wbc_ros/RadialPotentialField[] fields
